var/spack/repos/builtin/packages/cxx/package.py

```diff
@@ -14,28 +14,36 @@ class Cxx(Package):
     homepage = "https://isocpp.org/std/the-standard"
     virtual = True
 
-    def test(self):
+    def test_c(self):
```

Comment:
The name should match the compiler/class.

```suggestion
    def test_cxx(self):
```

Reply:
And the start of embedded test parts, which is correct below.